cobalt/worker/worker_global_scope.cc

```diff
@@ -178,8 +178,7 @@ class ScriptLoader : public base::MessageLoop::DestructionObserver {
     }
     if (content_type.empty()) {
       error->reset(new std::string(
-          base::StringPrintf(WorkerConsts::kServiceWorkerRegisterNoMIMEError,
-                             content_type.c_str())));
+          base::StringPrintf(WorkerConsts::kServiceWorkerRegisterNoMIMEError)));
```

Comment:
There was a print format error here - content_type is empty, so there's no reason to pass it to sprintf.